.cargo/config.toml

```diff
@@ -30,4 +30,6 @@ rustflags = [
   "-Aclippy::derivable_impls",           # false positives
   "-Aclippy::stable_sort_primitive",     # prefer stable sort
   "-Aclippy::extra-unused-type-parameters", # stylistic
+  "-Aclippy::default_constructed_unit_structs", # stylistic
+  "-Aclippy::bad-bit-mask",              # triggered by macro behaviour in bitflags 1.3.2
```

Comment:
Can we disable this one specifically in the files that need it?

Reply:
ohh right I forgot you can disable lints at a file level too, will do. (normal line-level lint bypasses ofc don't work on macro-generated code)